mobile/src/main/java/org/openhab/habdroid/model/Sitemap.kt

```diff
@@ -38,14 +38,12 @@ fun Node.toSitemap(): Sitemap? {
     var label: String? = null
     var name: String? = null
     var icon: String? = null
-    var link: String? = null
```

Comment:
It looks like the actual issue is that the 3rd parameter to the Sitemap construction below should be `link`, not `homepageLink`.

Reply:
The link property of `Sitemap` is only used in `SitemapTest#testGetLink()`. Should I remove `link` at all?

Reply:
I don't have a strong opinion there ... do whatever you prefer there :-)